pkg/vulnsrc/cargo/cargo.go

```diff
@@ -145,3 +150,53 @@ func (vs VulnSrc) Get(pkgName string) ([]Advisory, error) {
 	}
 	return results, nil
 }
+
+func parseAdvisoryMarkdown(src []byte) (codeBlock, title, description string) {
+	md := markdown.New()
+	tokens := md.Parse(src)
```

Comment:
`github.com/golang-commonmark/markdown` adds more dependencies. Less dependency is less problem. Can we go with the second way according to this comment? I believe an advisory consists of one code block and one section. I'm wondering if we can parse it ourselves. It looks a bit overkill in our case. If it is difficult, we can use the markdown library.
https://github.com/RustSec/advisory-db/issues/414#issuecomment-702197689

Reply:
No longer depend on github.com/golang-commonmark/markdown